### ISSUE TITLE
Events for newsletter subscription and enterprise demo request

### ISSUE THREAD

**Issue by christinang89** [Mashape/getkong.org#43]
1. For the enterprise form and the newsletter updates, when you call identify, please also add "created_at" = datetime (unix time)

2. For the enterprise form, on submit, please also call analytics.track event for “request_enterprise_demo" with the properties.
Example:
```
analytics.track('request_enterprise_demo',{
    company: 'Mashape',
    deployment: 'on-prem',
    email: 'christina+test3@mashape.com',
    name: 'christina test',
    phone: '4321',
    request_date: date/time of submission
})
```
3. For the newsletter updates, same thing, on submit, please call analytics.track event for “request_newsletter_updates” with the properties. 

Alternatively, you can use the trackform function instead of track, whichever one is easier for you - https://segment.com/docs/libraries/analytics.js/#track-form

Basically, if the same user (same email) submits the form twice, it should call the analytics.track event twice with two different submit date/time.
